docs/medical-api/more-info/webhooks.mdx

```diff
@@ -359,3 +359,47 @@ Example payload:
   ]
 }
 ```
+
+#### Bulk Document Download
+
+While you can download specified documents using the [get document url endpoint](/medical-api/api-reference/document/bulk-document-download), 
+patients can have a large number of documents, and it can be tedious to download one at a time. 
+
+Instead trigger a [bulk get document url](/medical-api/api-reference/document/bulk-document-download) query - one query per patient at any given time;
+When the result is ready, we’ll send a webhook message with all the downloadable urls for all the patients documents.
+Example payload:
+
+```json
+{
+"meta": {
+    "messageId": "bd4184dd-3b35-4c60-8130-447d1d528fea",
+    "when": "2023-11-30T05:05:12.215Z",
+    "type": "medical.document-bulk-download"
+  },
+  "patients": [
+    {
+      "status": "completed",
+      "documents": [
+        {
+          "id": "018c1b94-57dc-7950-b564-f74ab9b56cc4",
+          "size": 40670,
+          "status": "current",
+          "fileName": "81f2ec4e-c71c-4090-bf1c-9d8529f25b5a/018c1b94-2ed9-70f2-8b0f-ad4f8e8c7a16/81f2ec4e-c71c-4090-bf1c-9d8529f65b5a_018c1b94-2ed9-70f2-8b0f-ad4f8e8c7a16_018c1b94-57dc-7950-b564-f74ab9b56cc4.bin",
+          "mimeType": "application/xml",
+          "signedUrl": "<a-signed-url>"
+        },
+        {
+          "id": "018c1b94-57dc-7950-b564-f749bcf9be3e",
+          "size": 77093,
+          "status": "current",
+          "fileName": "81f2ec4e-c71c-4090-bf1c-9d8529f25b5a/018c1b94-2ed9-70f2-8b0f-ad4f8e8c7a16/81f2ec4e-c71c-4090-bf1c-9d8529f65b5a_018c1b94-2ed9-70f2-8b0f-ad4f8e8c7a16_018c1b94-57dc-7950-b564-f749bcf9be3e.bin",
+          "mimeType": "application/pdf",
```

Comment:
- why are the file extensions inconsistent with the mimeType?
- double checking that these are all pre-prod IDs?
- as a side note, why did we chose this arbitrary subset of properties to return for the document object? why not just make it consistent with `medical.document-download` - if we're going to be returning anything aside from just an array of URLs?
- `signedUrl` has no meaning to the cx based on what was previously discussed, how about use `url` to be consistent with the Download URL endpoint?

Reply:
1. that is a good point. I think that behavior should be solved by the mime type PR I have backlogged, but will change that. 
2. Yes + not real. 
3. Um I dont think medical document download spec has a simple intuitive way / place to put in the signed url. So I just included the information that I thought was relevant. 
4. done

Reply:
for 3:

what I was getting at is why don't we just include the exact same fields as the medical document download spec, but include the url for download

the shape will be more consistent with the other WH, just these fields seem a bit arbitrary